### PR TITLE
feat(inpage-nav): improve mobile keyboard navigation - FRONT-4501

### PR DESCRIPTION
**Pull request by emeryro**
- use arrows to open/close the inpage navigation
- focus first item on open

### CODE REVIEW

File: src/implementations/vanilla/components/inpage-navigation/inpage-navigation.js [hunk 1]
```diff
@@ -332,15 +334,15 @@ export class InpageNavigation {
       onDeactivate: () => this.closeList(),
     });
 
-    if (this.attachClickListener && toggleElement) {
-      toggleElement.addEventListener('click', this.handleClickOnToggler);
+    if (this.attachClickListener && this.toggleElement) {
+      this.toggleElement.addEventListener('click', this.handleClickOnToggler);
```
Comment (planctus) [inline src/implementations/vanilla/components/inpage-navigation/inpage-navigation.js:338]: did not look at the whole file but it seems that we are adding event listeners and not removing them in destroy() here

**Comment by planctus** [comment]
surely not because of this pr, but we have a flaw in the focus styles for the element, the label moves horizontally, maybe we can fix this here. Also the position of the icon seems not to be centered anymore ( in EC)

**Review by planctus — COMMENTED** [review]
i've just noticed a couple of issues related to the css, we can fix those here or just create a separate task for them, the scope of the pull request is already fullfilled

**Comment by emeryro** [comment]
let's handle this in another PR. This may be related to the scale/zoom setting, but on my side only the icon is "moving" with the focus, so to be investigated

**Comment by planctus** [comment]
Oh true, i forgot i had the scaling still active in my desktop ;)

**Pull request merged**